tests/mmptest/src/MMPTest.cs

```diff
@@ -688,7 +688,12 @@ public void MM2003 (string opt)
 		[Test]
 		public void BuildingSameSolutionTwice_ShouldNotRunACToolTwice ()
 		{
-			const string actool = " execution started with arguments: actool --errors --warnings --notices --output-format xml1 --output-partial-info-plist ";
+			Func<string, bool> executedActool = (line) => {
+				if (!line.Contains (" execution started with arguments"))
```

Comment:
why not `(lint) => line.Contains (" execution started with arguments") && line.Contains ("actool --errors --warnings --notices --output-format xml1 --output-partial-info-plist ");`?

Reply:
No reason, except that I wrote this on a tiny laptop with a small screen.

Reply:
Have you tried a smaller font? 😆